### PR TITLE
created cart page

### PR DESCRIPTION
**Pull request by Rockyraven**
created cart page
added funtionalitiy
 add remove cart button
add  move to wish list button
image

### CODE REVIEW

**Review by arjun-computer-geek — APPROVED** [review]
Everything is good. Folder structure.

File: src/context/WishContext.js [hunk 1]
```diff
@@ -4,16 +4,24 @@ const wishlistContext = createContext(null);
...
     const [ wishlist, setWishList ] = useState(JSON.parse(localStorage.getItem ("wishlist")));
+
     useEffect(()=> {
         localStorage.setItem("wishlist", JSON.stringify(wishlist))
     },[wishlist])
-
-  // funtions for add to cart and remove card  
-
+    
+    // funtions for add to cart and remove card
```
Comment (arjun-computer-geek) [inline src/context/WishContext.js]: This is good to have comments like this.

**Review by arjun-computer-geek — APPROVED** [review]
Else looks good

File: src/context/WishContext.js [hunk 2]
```diff
@@ -1,86 +1,119 @@
...
+          : item
+      )
+    );
+  };
+
+  const removeFromWishList = (id) => {
+    // for underStanding purpose
+    // let temproduct = wishlist
+    // temproduct = temproduct.filter(item => item._id !== id)
+    // setWishList(temproduct);
```
Comment (Shubhamkr2610) [inline src/context/WishContext.js:84]: remove unused code

**Review by arjun-computer-geek — APPROVED** [review]
looks good

**Review by Shubhamkr2610 — APPROVED** [review]
you are good to go

**Pull request merged**